streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorContextImpl.java

```diff
@@ -276,11 +276,6 @@ public V get(final K key) {
             return wrapped().all();
         }
 
-        @Override
-        public KeyValueIterator<K, V> prefixScan(final K prefix) {
-            return wrapped().prefixScan(prefix);
-        }
-
```

Comment:
It should be possible now to roll back all the "prefix" code. It was a good solution to the problem, but it's a pretty significant expansion to the KV store API, so it's better if we can avoid it.

Plus, it's a whole lot of code and tests that we don't need anymore. The code is very well written, but at the same time, every line of code in the project is a maintenence burden.

Reply:
Honestly I am thrilled to reduce the scope of changes. Just pleased as can be.